wallet/src/lib/rapyd/RapydClient.ts

```diff
@@ -0,0 +1,185 @@
+import axios from "axios";
+import * as Crypto from "crypto-js";
+import moment from "moment";
+import {
+  HttpMethods,
+  IRapydClient,
+  IRapydRequestParams,
+  RequestSignature,
+} from "./types";
+
+class RapydClient implements IRapydClient {
```

Comment:
This client is a great starting point to develop a Rapyd Typescript SDK in the future, which Rapyd developers would appreciate 🤔

Reply:
Yup, Developers using JS, TS and Node will love it.